template_feed/Microsoft.DotNet.Common.ProjectTemplates.7.0/content/ConsoleApplication-CSharp/.template.config/dotnetcli.host.json

```diff
@@ -16,6 +16,10 @@
     "langVersion": {
       "longName": "langVersion",
       "shortName": ""
+    },
+    "UseProgramMain": {
```

Comment:
For the [web project templates we use an `ide.host.json`](src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/.template.config/ide.host.json) to invert this when shown in Visual Studio's project creation dialog:

![image](https://user-images.githubusercontent.com/249088/160447954-5e75f794-e757-40ac-bc5f-881920e2e118.png)

Reply:
We don't use `ide.host.json` at the moment, and after discussing with @phenning won't do it for 6.0.3xx. 
I will try to introduce VS configuration, however as separate PR. 
Any configuration of `ide.host.json` won't be localized, so i'm not certain if we should go for inversion, having option unchecked by default seems good enough.

Reply:
Yeah I spoke with @phenning and he convinced me of the same. I'm going to undo that part of the web templates change.